lib/plugins/aws/package/compile/events/sns/index.js

```diff
@@ -136,57 +112,27 @@ class AwsCompileSNSEvents {
               } = event.sns.redrivePolicy;
               let targetArn;
               let targetUrl;
-              if (!deadLetterTargetArn && !deadLetterTargetRef && !deadLetterTargetImport) {
```

Comment:
It'll be probably worth to introduce a validation where we require just one definition form.

Such error if happens will come otherwise unnoticed (it's not a CF template properties that are mutually exclusive)

Reply:
I just saw in my PR that  I made them exclusively required :

```js
oneOf: [
                { required: ['deadLetterTargetArn'] },
                { required: ['deadLetterTargetRef'] },
                { required: ['deadLetterTargetImport'] },
              ],
```

Just now updating `oneOf` to `anyOf`

Reply:
Indeed `oneOf` should work great, thanks for clarifying 👍